classes/manager.php

```diff
@@ -16,13 +16,11 @@
 
 namespace tool_excimer;
 
-defined('MOODLE_INTERNAL') || die();
-
 /**
  * Primary controller class for handling Excimer profiling.
  *
  * @package   tool_excimer
- * @author    Jason den Dulk <jasondendulk@catalyst-au.net>
+ * @author    Jason den Dulk <jasondendulk@catalyst-au.net>, Kevin Pham <kevinpham@catalyst-au.net>
```

Comment:
super minor, but I think this works better generally as 2 `@author` lines

Reply:
done